tests/foreman/cli/test_user.py

```diff
@@ -375,7 +375,7 @@ def test_positive_create_ssh_key_super_admin_from_file(self, ssh_key, default_sa
         :CaseImportance: Critical
         """
         ssh_name = gen_string('alpha')
-        result = default_sat.execute(f'''echo '{ssh_key}' > test_key.pub''')
+        result = default_sat.execute(f"echo '{ssh_key}' > test_key.pub")
```

Comment:
didn't we want to use it: with single quotes outer and double inside ?

Reply:
So the order of preference is single quote, double quote, triple single, triple double, then escapes.
```
'this'
"this 'nested'"
'''even "more 'nesting' here"'''
"""we ''' should " hopefully ' never ' get " this ''' far"""
```